.github/workflows/report_jokes/main.py

```diff
@@ -0,0 +1,41 @@
+import json
+import urllib3
+
+# documentation available here : https://sv443.net/jokeapi/v2/
```

Comment:
We may change in the future the source api, we should find a way to link a resource api to it's response schema more dynamically !

could be interesting !